excellent/functions/functions.go

```diff
@@ -202,6 +203,21 @@ func Array(env utils.Environment, args ...types.XValue) types.XValue {
 	return types.NewXArray(args...)
 }
 
+// Number tries to convert `value` to a number. An error is returned if the value can't be converted.
+//
+//   @(number(10)) -> 10
+//   @(number("123.45000")) -> 123.45
+//   @(number("what?")) -> ERROR
+//
+// @function number(value)
+func Number(env utils.Environment, value types.XValue) types.XValue {
```

Comment:
was going to call this `to_number` but `number` seems more natural and is consistent with `date` which essentially does the same thing

Reply:
I'm ok with that but then we lose the symmetry of `from_` and `to_`. What are we going to call the thing that parses JSON?

Reply:
I don't forsee that being a commonly function since JSON typically enters excellent via something like `run.webhook.json` and I just had to look at the code because I couldn't remember what it returns (JSON from a string). I think I prefer something like `parse_json`.

Reply:
works for me.